### PR TITLE
Connection front end

### PR DESCRIPTION
**Pull request by LonelySam**
We have the request of GET and POST working with the FrontEnd. Howerer, the connection with the database is still in progress

### CODE REVIEW

File: src/SchemaGameDB.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const Sequelize = require('sequelize');
+
+function generateSchemaGame(instanceSequalize) {
```
Comment (pabloorellana) [inline src/SchemaGameDB.js]: it should be "generateGameSchema" or "buildGameSchema" and"sequalizeInstance"

File: src/Game.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		if(game === undefined) {
+			return Promise.reject()
+		}
+		return Promise.resolve({
+			id : game.id,
+			playerId : idHelper()
+		});
+	}
+}
+
```
Comment (pabloorellana) [inline src/Game.js]: I think most of the implementation is missing yet...

**Review by pabloorellana — CHANGES_REQUESTED** [review]

File: src/SchemaGameDB.js (hunk 1 above)
Comment (LonelySam) [reply src/SchemaGameDB.js]: Changed the name of the file and variables

File: src/Game.js (hunk 2 above)
Comment (LonelySam) [reply src/Game.js]: Finished the implementation

File: Index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    .authenticate()
+    .then(() => {
+      console.log('Connection has been established successfully.');
+      return generateGameSchema(connectionDB)
+        .sync({force: false})
+    })
+    .then(() => console.log("Creation of table Game succesful"))
+    .catch(err => {
+      console.error('Unable to connect to the database:', err);
+    });
```
Comment (pabloorellana) [inline Index.js]: remember to refactor this, the server should start only after the database connection has been established:
```javascript
 connectionDB
   .authenticate()
   .then(() => {
      console.log('Connection has been established successfully.');
      return generateGameSchema(connectionDB).sync({force: false})
   })
   .then(() => {
     console.log("Creation of table Game succesful")
     app.listen(3000, () => {
       console.log('Example app listening on port 3000!') 
     });
   })
   .catch(err => {
      console.error('Unable to connect to the database:', err);
   });
```

or you can implement your wrapper for the "server" object and to have something more elegant like:
```javascript
DatabaseConnection.init()
  .then(() => {
    return Server.init({ port: 3000 })
  })
  .then(() => {
    console.log('Example app listening on port 3000!')
  })
  .catch(err => {
    console.error('Unable to start the application', err);
  });

```

File: src/Game.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+					adversary: idAdversary
+				});
+			})
+			.then(savedGame => savedGame.dataValues)
+			.catch(error => {
+				console.log(error);
+				throw error;
+			});
+	}
+}
```
Comment (pabloorellana) [inline src/Game.js:48]: please fis your indentations

**Review by pabloorellana — APPROVED** [review]

**Comment by pabloorellana** [comment]
Please fix these last comments, after that you can merge this PR by yourself and continue working in the next tasks

File: src/Game.js (hunk 4 above)
Comment (LonelySam) [reply src/Game.js:48]: I fixed, but I think that of pushing the file it doesn't change properly.

File: Index.js (hunk 3 above)
Comment (LonelySam) [reply Index.js]: I already separated the start up of the application in different files

**Pull request merged**